File: 344-reverse-string/344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        
        int sLen = s.size();
        int halfLen = sLen / 2;
        
        for (int i = 0; i < halfLen; i++) {
            
            char tmp = s[i];
            s[i] = s[sLen - 1 - i];
            s[sLen - 1 - i] = tmp;
        }
    }
};

File: 344-reverse-string/344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        
        sLen = len(s)
        
        for i in range(sLen // 2):
            s[i], s[sLen - 1 - i] = s[sLen - 1 - i], s[i]
            
        